### ISSUE TITLE
An ability to use standard window frame

### ISSUE THREAD

**Issue by vrubleg** [telegramdesktop/tdesktop#2547]
Please make an ability to use default Windows window frame. The custom window frame of the Telegram Desktop looks ugly and works buggy.
### Expected behaviour

It have to work like all the other windows in the OS. The window have to have a border, just because all other windows also have it. And when I disable all shadows for all windows in the system settings, Telegram have to follow this setting also. I would like to have a normal border and no shitty shadows under the window.
### Actual behaviour

Actual implementation doesn't follow the system settings and, in fact, it is buggy:
1. When we maximizing the window, at first the shadow of the window is shown, and only after it the window flies out to its place from the taskbar.
2. When I'm switching to another virtual desktop (using Virtuoz), the window of the Telegram will be hidden, but the shadow of this window remains to be displayed on all virtual desktops.
3. Sometimes this shadow isn't created (I can't reproduce it easily). The window doesn't have border at all, and and as the result it may look like this (the window of the Telegram is above the window of the Skype): http://veg.by/z/2016-10-07-00-23-06-426bfa9f.png
### Configuration

**Operating system:** Windows 7.

**Version of Telegram Desktop:** 0.10.16.

**Comment by frozenspider** [comment]
+1, I would additionally request an option for Telegram to use the default system style - normal buttons, forms, pop-up menus (including the one appearing when right-clicking tray icon).
All these customized styles and swipe-menus are useless and inconvenient on desktop client.

**Comment by navossoc** [comment]
Any news on this!?

**Comment by vrubleg** [comment]
Additional problem with non-standard frame: it is not clear whether the window is active or not.

**Comment by navossoc** [comment]
@vrubleg Exactly, it is pretty annoying...
I have just installed the app. The first thing I looked for is some setting to disable the theme, but I don't think it exists.

**Comment by stek29** [comment]
Closing in favor of #2958
